act-rules-format.bs

```diff
@@ -89,17 +89,40 @@ An ACT Rule MUST have a description that is in plain language and provides a bri
 Accessibility Requirements {#structure-accessibility-requirements}
 ==================================================================
 
-<div class=note>
-  **Editor's note:** The ACT Task Force is looking for feedback about the use of the term "pass" in relation to rules. While rules can "pass", their corresponding accessibility requirement can fail. A section has been added to make this explicit in the rule, but we would like to know if this is sufficient.
+Accessibility requirements are just that: A requirement that a particular web page must conform to for it to be considered accessible. This can (and usually does) include the WCAG success criteria. Often organizations have additional requirements which may come from different sources, such as laws, internal standards, etc. These too are considered accessibility requirements and can be tested using the ACT Rules Format. What the precise requirements are for any test is beyond the scope of the ACT Rules Format.
```

Comment:
- "a particular web page" -> "particular content" (to make it less page-centric, for example documents, web or mobile applications, etc.)
- "Often" -> "Some" (I think "often" is implying a level of frequency that is hard to prove, whereas "some" is more open)
- remove "which may come" (to get rid of "may" keyword)
- "are considered" -> "can be considered" (qualifier needed IMO)

Reply:
I don't think "particular" needs to be in there either, so it could simply say "content".  

I agree with your other comments on this part.